druid/src/text/attribute.rs

```diff
@@ -179,6 +181,16 @@ impl AttributeSpans {
         items.sort_by(|a, b| a.0.start.cmp(&b.0.start));
         items
     }
+
+    pub(crate) fn env_update(&self, ctx: &EnvUpdateCtx) -> bool {
+        self.size
+            .iter()
+            .any(|span_attr| ctx.env_key_changed(&span_attr.attr))
+            || self
+                .fg_color
+                .iter()
+                .any(|span_attr| ctx.env_key_changed(&span_attr.attr))
```

Comment:
do we also want to do `font_descriptor`?

Reply:
missed that one